### ISSUE TITLE
[Refactor] .setAttribute should be removed and use 'aria-label' prop instead

### ISSUE THREAD

**Issue by ketsappi** [vrk-kpa/suomifi-design-system#87]
### Problem

[src/pages/components/button.tsx](https://github.com/vrk-kpa/suomifi-design-system/blob/a326a3686bec15242bda7f83ab086ea20ab03df6/src/pages/components/button.tsx) uses .setAttribute to change the components 'aria-label'. 

https://github.com/vrk-kpa/suomifi-design-system/blob/a326a3686bec15242bda7f83ab086ea20ab03df6/src/pages/components/button.tsx#L61

### Solution suggestion

> Component should get the changed aria-label by updating prop 'aria-label'.

As was discussed on the PR #86

**Comment by ketsappi** [comment]
Closed as the mentioned part, `setAttribute` is no longer present on the file.
